sklearn/feature_extraction/text.py

```diff
@@ -87,6 +88,8 @@ def strip_tags(s):
 
 def _check_stop_list(stop):
     if stop == "english":
+        warnings.warn("stop_words='english' is deprecated in version 0.20 "
+                      "and will be removed in 0.22.", DeprecationWarning)
```

Comment:
There should be some suggested action here.  Maybe,
```
[..] will be removed in 0.22. Prove the full stop word list or
consider using the max_df parameter, if possible.
```
?

Reply:
Will be done. In next commit :+1:

Reply:
Done